.travis.yml

```diff
@@ -4,10 +4,16 @@ script:
   - cd Examples
   - xcodebuild -version
   - xcodebuild -project DynamicColorExample.xcodeproj -scheme iOSTests -sdk iphonesimulator -destination "platform=iOS Simulator,name=iPhone 6" -configuration Release ONLY_ACTIVE_ARCH=YES -enableCodeCoverage YES test
-  - cd $(xcodebuild -project DynamicColorExample.xcodeproj -showBuildSettings | grep 'TEMP_ROOT = [/]' | grep -Eo '/(.*?)$')
-  - cd CodeCoverage
-  - xcrun llvm-cov report -instr-profile Coverage.profdata Products/Release-iphonesimulator/iOSTests.xctest/iOSTests
-  - cd $TRAVIS_BUILD_DIR/Examples
-  - bash <(curl -s https://codecov.io/bash) -cF ios -J '^DynamicColor$'
+  # - cd $(xcodebuild -project DynamicColorExample.xcodeproj -showBuildSettings | grep 'TEMP_ROOT = [/]' | grep -Eo '/(.*?)$')
+  - xcodebuild -showBuildSettings 2>/dev/null | grep -i "^\s*PRODUCT_NAME" | sed -e 's/.*= \(.*\)/\1/'
```

Comment:
This is because you are not in the right directory and it misses the `-project` parameter.

With this:
```bash
$ xcodebuild -project DynamicColorExample.xcodeproj -showBuildSettings 2>/dev/null | grep -i "^\s*PRODUCT_NAME" | sed -e 's/.*= \(.*\)/\1/'
```

It gives me the good result:
```
DynamicColor
```